eng/versioning/version_client.txt

```diff
@@ -88,8 +88,8 @@ com.azure:azure-cosmos;4.33.1;4.34.0-beta.1
 com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
 com.azure:azure-cosmos-dotnet-benchmark;4.0.1-beta.1;4.0.1-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3_2-12;1.0.0-beta.1;1.0.0-beta.1
-com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12;4.12.1;4.13.0-beta.1
-com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12;4.12.1;4.13.0-beta.1
+com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12;4.12.1;4.12.2
```

Comment:
Are we planning to ship this release with azure-cosmos 4.34.0-beta.1 version?

Reply:
yes - shaded - beta tag will be renamed to snapshot. And because we shade azure-cosmos we don't also have to release it.